The Dart VM service is listening on http://127.0.0.1:58390/Ad2tCAnQ548=/
The Dart DevTools debugger and profiler is available at: http://127.0.0.1:58390/Ad2tCAnQ548=/devtools?uri=ws://127.0.0.1:58390/Ad2tCAnQ548=/ws
The Dart VM service is listening on http://127.0.0.1:61508/t8t3MhWbTvQ=/
The Dart DevTools debugger and profiler is available at: http://127.0.0.1:61508/t8t3MhWbTvQ=/devtools?uri=ws://127.0.0.1:61508/t8t3MhWbTvQ=/ws
The Dart VM service is listening on http://127.0.0.1:65330/DI3utFwse2w=/
The Dart DevTools debugger and profiler is available at: http://127.0.0.1:65330/DI3utFwse2w=/devtools?uri=ws://127.0.0.1:65330/DI3utFwse2w=/ws
The Dart VM service is listening on http://127.0.0.1:65087/IDur0OET5Fk=/
The Dart DevTools debugger and profiler is available at: http://127.0.0.1:65087/IDur0OET5Fk=/devtools?uri=ws://127.0.0.1:65087/IDur0OET5Fk=/ws
The Dart VM service is listening on http://127.0.0.1:52126/1BNcHDI8rYU=/
The Dart DevTools debugger and profiler is available at: http://127.0.0.1:52126/1BNcHDI8rYU=/devtools?uri=ws://127.0.0.1:52126/1BNcHDI8rYU=/ws
The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
Failed to read pubspec.yaml at D:\FlutterWorkspace\all_flutter_widget:
ParsedYamlException: line 32, column 19: Expected ':'.
   ╷
32 │   flutter_riverpod: ^2.6.1
   │                   ^
   ╵
#0      checkedYamlDecode (package:checked_yaml/checked_yaml.dart:32:5)
#1      new Pubspec.parse (package:pubspec_parse/src/pubspec.dart:185:7)
#2      parsePubspec (package:custom_lint_core/src/package_utils.dart:85:18)
<asynchronous suspension>
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:910:28)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:505:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:913:7)
#1      _rootRunBinary (dart:async/zone.dart:1438:47)
#2      _CustomZone.runBinary (dart:async/zone.dart:1331:19)
#3      _FutureListener.handleError (dart:async/future_impl.dart:219:22)
#4      Future._propagateToListeners.handleError (dart:async/future_impl.dart:914:47)
#5      Future._propagateToListeners (dart:async/future_impl.dart:935:13)
#6      Future._completeError (dart:async/future_impl.dart:715:5)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\riverpod-2.6.1\lib\src\async_notifier.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:507:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:928:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\riverpod-2.6.1\lib\src\builders.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:507:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:928:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\bottom_navigation_bar_item.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_view.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\framework.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_view.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\framework.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\framework.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_view.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request analysis.setContextRoots failed with the following error:
RequestErrorCode.PLUGIN_ERROR
The plugin extended_image was not found in the package config at D:\FlutterWorkspace\all_flutter_widget. Make sure to run `pub get` first.
at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:931:11)
#1      MappedIterator.moveNext (dart:_internal/iterable.dart:413:20)
#2      Future.wait (dart:async/future.dart:518:26)
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:923:34)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:500:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request analysis.setSubscriptions failed with the following error:
RequestErrorCode.PLUGIN_ERROR
The plugin extended_image was not found in the package config at D:\FlutterWorkspace\all_flutter_widget. Make sure to run `pub get` first.
at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:931:11)
#1      MappedIterator.moveNext (dart:_internal/iterable.dart:413:20)
#2      Future.wait (dart:async/future.dart:518:26)
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:923:34)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:500:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request analysis.setPriorityFiles failed with the following error:
RequestErrorCode.PLUGIN_ERROR
The plugin extended_image was not found in the package config at D:\FlutterWorkspace\all_flutter_widget. Make sure to run `pub get` first.
at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:931:11)
#1      MappedIterator.moveNext (dart:_internal/iterable.dart:413:20)
#2      Future.wait (dart:async/future.dart:518:26)
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:923:34)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:500:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
The plugin extended_image was not found in the package config at D:\FlutterWorkspace\all_flutter_widget. Make sure to run `pub get` first.
at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:931:11)
#1      MappedIterator.moveNext (dart:_internal/iterable.dart:413:20)
#2      Future.wait (dart:async/future.dart:518:26)
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:923:34)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:500:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
The plugin extended_image was not found in the package config at D:\FlutterWorkspace\all_flutter_widget. Make sure to run `pub get` first.
at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:931:11)
#1      MappedIterator.moveNext (dart:_internal/iterable.dart:413:20)
#2      Future.wait (dart:async/future.dart:518:26)
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:923:34)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:500:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request analysis.updateContent failed with the following error:
RequestErrorCode.PLUGIN_ERROR
The plugin extended_image was not found in the package config at D:\FlutterWorkspace\all_flutter_widget. Make sure to run `pub get` first.
at:
#0      CustomLintProject.parse.<anonymous closure> (package:custom_lint/src/workspace.dart:931:11)
#1      MappedIterator.moveNext (dart:_internal/iterable.dart:413:20)
#2      Future.wait (dart:async/future.dart:518:26)
#3      CustomLintProject.parse (package:custom_lint/src/workspace.dart:923:34)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
<asynchronous suspension>
#5      CustomLintWorkspace.fromContextRoots (package:custom_lint/src/workspace.dart:500:22)
<asynchronous suspension>
#6      SocketCustomLintServerToClientChannel.create (package:custom_lint/src/v2/server_to_client_channel.dart:54:23)
<asynchronous suspension>
#7      CustomLintServer._maybeSpawnCustomLintPlugin (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:360:23)
<asynchronous suspension>
#8      CustomLintServer._handleAnalysisSetContextRoots.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:341:9)
<asynchronous suspension>
#9      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#10     CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

[WrapWithConsumer] 2025-03-28T21:49:34.963309 Plugin WrapWithConsumer threw while analyzing D:\FlutterWorkspace\all_flutter_widget\lib\complex_widget\ListViewListener.dart:
[WrapWithConsumer] 2025-03-28T21:49:34.963309 InconsistentAnalysisException: Requested result might be inconsistent with previously returned results
[WrapWithConsumer] 2025-03-28T21:49:34.963309
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #0      AnalysisSessionImpl.checkConsistency (package:analyzer/src/dart/analysis/session.dart:61:7)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #1      AnalysisSessionImpl.getResolvedUnit (package:analyzer/src/dart/analysis/session.dart:160:5)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #2      ChangeBuilderImpl._createDartFileEditBuilder (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:320:33)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #3      ChangeBuilderImpl.addDartFileEdit (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:153:23)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #4      ChangeBuilderImpl.addDartFileEdit.<anonymous closure> (package:custom_lint_core/src/change_reporter.dart:241:35)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #5      new Future.<anonymous closure> (dart:async/future.dart:258:40)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #6      _rootRun (dart:async/zone.dart:1406:47)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #7      _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #8      _CustomZone.runGuarded (dart:async/zone.dart:1225:7)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #9      _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1265:23)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #10     _rootRun (dart:async/zone.dart:1414:13)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #11     _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #12     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1249:23)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #13     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #14     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #15     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
[WrapWithConsumer] 2025-03-28T21:49:34.963309 #16     _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
[WrapWithConsumer] 2025-03-28T21:49:34.963309
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 Plugin WrapWithProviderScope threw while analyzing D:\FlutterWorkspace\all_flutter_widget\lib\complex_widget\ListViewListener.dart:
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 InconsistentAnalysisException: Requested result might be inconsistent with previously returned results
[WrapWithProviderScope] 2025-03-28T21:49:34.963309
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #0      AnalysisSessionImpl.checkConsistency (package:analyzer/src/dart/analysis/session.dart:61:7)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #1      AnalysisSessionImpl.getResolvedUnit (package:analyzer/src/dart/analysis/session.dart:160:5)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #2      ChangeBuilderImpl._createDartFileEditBuilder (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:320:33)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #3      ChangeBuilderImpl.addDartFileEdit (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:153:23)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #4      ChangeBuilderImpl.addDartFileEdit.<anonymous closure> (package:custom_lint_core/src/change_reporter.dart:241:35)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #5      new Future.<anonymous closure> (dart:async/future.dart:258:40)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #6      _rootRun (dart:async/zone.dart:1406:47)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #7      _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #8      _CustomZone.runGuarded (dart:async/zone.dart:1225:7)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #9      _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1265:23)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #10     _rootRun (dart:async/zone.dart:1414:13)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #11     _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #12     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1249:23)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #13     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #14     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #15     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309 #16     _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
[WrapWithProviderScope] 2025-03-28T21:49:34.963309
The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\extended_image-9.1.0\lib\src\extended_image.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\extended_image-9.1.0\lib\src\extended_image.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\extended_image-9.1.0\lib\src\extended_image.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\extended_image-9.1.0\lib\src\extended_image.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\extended_image-9.1.0\lib\src\extended_image.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to C:\Users\huangyuan\AppData\Local\Pub\Cache\hosted\mirrors6.tuna.tsinghua.edu.cn%47dart-pub%47\extended_image-9.1.0\lib\src\extended_image.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\scheduler\binding.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

[WrapWithConsumer] 2025-03-28T22:15:17.350627 Plugin WrapWithConsumer threw while analyzing D:\FlutterWorkspace\all_flutter_widget\lib\complex_widget\ListViewListener.dart:
[WrapWithConsumer] 2025-03-28T22:15:17.350627 InconsistentAnalysisException: Requested result might be inconsistent with previously returned results
[WrapWithConsumer] 2025-03-28T22:15:17.350627
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #0      AnalysisSessionImpl.checkConsistency (package:analyzer/src/dart/analysis/session.dart:61:7)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #1      AnalysisSessionImpl.getResolvedUnit (package:analyzer/src/dart/analysis/session.dart:160:5)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #2      ChangeBuilderImpl._createDartFileEditBuilder (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:320:33)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #3      ChangeBuilderImpl.addDartFileEdit (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:153:23)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #4      ChangeBuilderImpl.addDartFileEdit.<anonymous closure> (package:custom_lint_core/src/change_reporter.dart:241:35)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #5      new Future.<anonymous closure> (dart:async/future.dart:258:40)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #6      _rootRun (dart:async/zone.dart:1406:47)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #7      _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #8      _CustomZone.runGuarded (dart:async/zone.dart:1225:7)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #9      _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1265:23)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #10     _rootRun (dart:async/zone.dart:1414:13)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #11     _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #12     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1249:23)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #13     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #14     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #15     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
[WrapWithConsumer] 2025-03-28T22:15:17.350627 #16     _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
[WrapWithConsumer] 2025-03-28T22:15:17.350627
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 Plugin WrapWithProviderScope threw while analyzing D:\FlutterWorkspace\all_flutter_widget\lib\complex_widget\ListViewListener.dart:
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 InconsistentAnalysisException: Requested result might be inconsistent with previously returned results
[WrapWithProviderScope] 2025-03-28T22:15:17.351631
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #0      AnalysisSessionImpl.checkConsistency (package:analyzer/src/dart/analysis/session.dart:61:7)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #1      AnalysisSessionImpl.getResolvedUnit (package:analyzer/src/dart/analysis/session.dart:160:5)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #2      ChangeBuilderImpl._createDartFileEditBuilder (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:320:33)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #3      ChangeBuilderImpl.addDartFileEdit (package:analyzer_plugin/src/utilities/change_builder/change_builder_core.dart:153:23)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #4      ChangeBuilderImpl.addDartFileEdit.<anonymous closure> (package:custom_lint_core/src/change_reporter.dart:241:35)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #5      new Future.<anonymous closure> (dart:async/future.dart:258:40)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #6      _rootRun (dart:async/zone.dart:1406:47)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #7      _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #8      _CustomZone.runGuarded (dart:async/zone.dart:1225:7)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #9      _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1265:23)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #10     _rootRun (dart:async/zone.dart:1414:13)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #11     _CustomZone.run (dart:async/zone.dart:1317:19)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #12     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1249:23)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #13     Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #14     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #15     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631 #16     _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
[WrapWithProviderScope] 2025-03-28T22:15:17.351631
The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_notification.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_notification.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_notification.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

The request edit.getAssists failed with the following error:
RequestErrorCode.PLUGIN_ERROR
A request threw the exception:Bad state: Unable to find the context to D:\flutter\packages\flutter\lib\src\widgets\scroll_notification.dart
#0      AnalysisContextCollectionImpl.contextFor (package:analyzer/src/dart/analysis/analysis_context_collection.dart:145:5)
#1      _ClientAnalyzerPlugin.handleEditGetAssists (package:custom_lint_builder/src/client.dart:509:47)
<asynchronous suspension>
#2      ServerPlugin._getResponse (package:analyzer_plugin/plugin/plugin.dart:554:18)
<asynchronous suspension>
#3      ServerPlugin._onRequest (package:analyzer_plugin/plugin/plugin.dart:597:18)
<asynchronous suspension>

at:
#0      SocketCustomLintServerToClientChannel.sendCustomLintRequest.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:268:11)
#1      _$CustomLintResponseAnalyzerPluginResponseImpl.map (package:custom_lint/src/v2/protocol.freezed.dart:967:34)
#2      SocketCustomLintServerToClientChannel.sendCustomLintRequest (package:custom_lint/src/v2/server_to_client_channel.dart:262:14)
<asynchronous suspension>
#3      SocketCustomLintServerToClientChannel.sendAnalyzerPluginRequest (package:custom_lint/src/v2/server_to_client_channel.dart:234:22)
<asynchronous suspension>
#4      CustomLintServer._handleRequest.<anonymous closure>.<anonymous closure> (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:190:17)
<asynchronous suspension>
#5      PendingOperation.run (package:custom_lint/src/async_operation.dart:22:14)
<asynchronous suspension>
#6      CustomLintServer._handleRequest (package:custom_lint/src/v2/custom_lint_analyzer_plugin.dart:173:22)
<asynchronous suspension>

